k8s/kubectl/admin/yaml/example/qserv-su.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Maybe add a comment block to the top of this file explaining what the example does and how to use?